rclpy/src/rclpy/_rclpy.c

```diff
@@ -221,7 +221,10 @@ rclpy_remove_ros_args(PyObject * Py_UNUSED(self), PyObject * args)
         for (int ii = 0; ii < nonros_argc; ++ii) {
           PyList_SET_ITEM(result_list, ii, PyUnicode_FromString(nonros_argv[ii]));
         }
+#pragma warning(push)
+#pragma warning(disable: 4090)
```

Comment:
Maybe post a link to the warning suppression justification in a comment?

https://github.com/ros2/rclpy/pull/180#issuecomment-375452757

Reply:
Put it in the commit message, not in the source, but that's a good idea :)